pipeline.yml

```diff
@@ -3,27 +3,27 @@ env:
   - SLACK_QUANTUM_BUILD_CHANNEL : "#dynamo-jenkinsbuild"
   - SLACK_QUANTUM_BUILD_CREDENTIAL_ID : "slack-notify-token"
   - MAIL_QUANTUM_BUILD_RECIPIENT : "dynamo.plaid.dev@autodesk.com"
+  - ARTIFACTORY_BOBCAT_DOCKER_REGISTRY : "artifactory.dev.adskengineer.net"
   - TEST_REPORT_FORMAT : "junit"
 
 check_changelog_updated_on_pr: false
 pipeline_os: "Linux"
 force_slack_notification : true
-language: typescript
 nodejs_version: "10"
 
 build:
   customized_builder_image:
     registry: artifactory
-    url: autodesk-docker.art-bobcat.autodesk.com/quantum/hardened-nodejs:latest
+    url: artifactory.dev.adskengineer.net/autodeskcloud/cloudosv2-base-nodejs-10:latest
   before_scripts:
-    - "npm install"
+    - "sudo npm install --unsafe-perm=true"
```

Comment:
Why do we need to perform this change here?

Reply:
We are having difficulties executing the npm install inside the docker container.

![image](https://user-images.githubusercontent.com/48030764/65182816-c11d1880-da27-11e9-85a2-2c5053c3667a.png)

We find this solution 

https://stackoverflow.com/questions/35954725/error-eacces-permission-denied-when-trying-to-install-eslint-using-npm

Or reinstall npm

https://docs.npmjs.com/resolving-eacces-permissions-errors-when-installing-packages-globally

Reply:
@pinzart @jasonstratton  should this be necessary to install npm?

Reply:
I commented on slack regarding this issue

Reply:
We already removed the sudo instruction.